core/controllers/topic_editor.py

```diff
@@ -271,9 +250,6 @@ def put(self, topic_id):
 
         topic_domain.Topic.require_valid_topic_id(topic_id)
         topic = topic_services.get_topic_by_id(topic_id, strict=False)
-        if topic is None:
```

Comment:
Ditto, add check for topic also to decorator.

Reply:
Done

Reply:
Reverted the changes as its gonna make the next few lines untested

Reply:
Ditto

Reply:
Done